calico_kubernetes/calico_kubernetes.py

```diff
@@ -67,13 +73,14 @@ def delete(self, namespace, pod_name, docker_id):
         self.pod_name = pod_name
         self.docker_id = docker_id
         self.namespace = namespace
+        workload_id = self._get_container_id(self.docker_id)
```

Comment:
I'm pretty sure we don't need to call _get_container_id here - self.docker_id should already be the container ID, as far as I know.